packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -137,6 +137,7 @@ class InteractionResponses {
   }
 
   /**
+   * Options that can be passed into editReply.
```

Comment:
I think it's better to link the method rather than just mentioning it, at first, you can try this:
```suggestion
   * Options that can be passed into {@link editReply}.
```

If the website doesn't pick the link up, I suppose you can fall back to the following option:
```suggestion
   * Options that can be passed into {@link InteractionResponses.editReply}.
```